website/content/docs/get-started/developer-qs.mdx

```diff
@@ -167,6 +167,68 @@ import org.springframework.vault.core.VaultTemplate;
 
 </CodeBlockConfig>
 
+</Tab>
+<Tab heading="OpenAPI Go (Beta)" group="openAPI-go">
+
+
+[OpenAPI Go](https://github.com/hashicorp/vault-client-go) (Beta) client library:
+
+```shell-session
+$ export GOPRIVATE=github.com/hashicorp/vault-client-go
+$ go get github.com/hashicorp/vault-client-go
+```
+
+Now, let's add the import statements for the client library to the top of the file.
+
+<CodeBlockConfig heading="import statements for client library" lineNumbers>
+
+```go
+import (
+	"github.com/hashicorp/vault-client-go"
+	"github.com/hashicorp/vault-client-go/schema"
+)
+```
+
+</CodeBlockConfig>
+
+
+</Tab>
+<Tab heading="OpenAPI C# (Beta)" group="openAPI-cs">
+
+
+[OpenAPI C#](https://github.com/hashicorp/vault-client-dotnet) (Beta) client library:
+
+The Nuget packages are hosted in an internal Nuget feed that can be found in [Artifactory](https://artifactory.hashicorp.engineering/ui/repos/tree/General/vault-devex-nuget-local). You can use the Dotnet CLI or Nuget CLI to retrieve the package.
+
+You need to add the private Nuget feed as a source in either CLI. You can also generate an [access token](https://www.jfrog.com/confluence/display/JFROG/User+Profile#UserProfile-IdentityTokenidentitytoken) that can be used in lieu of a password.
+
+```shell-session
+$ nuget sources add \
```

Comment:
Some of this won't be required when we publish directly to Nuget instead of Artifactory. Updated instructions for this can be found in this [PR](https://github.com/hashicorp/vault-client-dotnet/pull/86)